temporian/core/processor.py

```diff
@@ -139,15 +175,24 @@ def infer_processor(inputs: List[Event], outputs: List[Event]) -> Preprocessor:
 
         else:
             p.operators().add(feature.creator())
-            # Add the parent features
+            p.samplings().add(feature.sampling())
```

Comment:
Nit: would change these to `add_operator` and `add_sampling` methods in `Processor` for consistency with methods in operator class

Reply:
same with `add_event` below

Reply:
Good point.

*Added and used add_operator, add_sampling, add_feature and add_event.